scripts/build-types.js

```diff
@@ -0,0 +1,10 @@
+import fs from "fs";
+
+function copyFile(sourcePath, destinationPath) {
```

Comment:
nitpick: calling `fs.copyFileSync` directly is simpler than extracting out a reusable copyFile function. Please remove.

Reply:
Removed the script, as it is no longer needed.